routes/web.php

```diff
@@ -6,27 +6,53 @@
 |--------------------------------------------------------------------------
 | Web Routes
 |--------------------------------------------------------------------------
-|
 | Here is where you can register web routes for your application. These
 | routes are loaded by the RouteServiceProvider within a group which
-| contains the "web" middleware group. Now create something great!
-|
+| contains the "web" middleware group.
 */
 
-Route::get('/', function () {
-    return 'Home Page';
-})->name('home.index');
-
-// example route that we added as part of the tutorial
-Route::get('/contact', function() {
-    return 'Contact';
-})->name('home.contact');
-
-// example route to show the power of parameters (optional as well with ? + default)
-Route::get('/month/{id?}', function($id = '5') {
-    return 'Your month id is: ' . $id;
-})
-// ->where([
-//     'id' => '[0-9]+'
-// ])
-->name('home.month.monthId');
+// ROUTE:  home
+// DESC:   returns the main page of the application
+Route::view('/', 'home',  ['name' => 'Rob']
+)->name('view.home');
+
+// ROUTE:  category
+// METHOD: get, post
+// PARAMS: id (optional)
+Route::match(['get', 'post'], '/category/{id?}', function ($id=null) {
```

Comment:
Route::match() -- this is how you can match multiple methods with the same route path.